src/scheduler/period-validity.directive.spec.js

```diff
@@ -0,0 +1,94 @@
+describe('Period validity Directive', function() {
+    var element,
+        scope,
+        form;
+
+    beforeEach(module('motech-scheduler'));
+
+    beforeEach(inject(function ($compile, $rootScope, $httpBackend) {
+        $httpBackend.when('GET', '/server/status/status.html').respond({});
+        $httpBackend.when('GET', 'http://localhost:8080/motech-platform-server/server/bootstrap/status').respond({});
+
+        scope = $rootScope.$new();
+        element = angular.element(
+            '<form name="form">' +
+                '<input type="text" ng-model="model.period" name="period" period-validity />' +
+            '</form>'
+        );
+        scope.model = { period: null };
+        $compile(element)(scope);
+        scope.$digest();
+        form = scope.form;
+    }));
+
+    describe('string', function() {
```

Comment:
Shouldn't it be something different than 'string'? I believe that we can delete this describe line and leave the test as they are